torch/csrc/autograd/variable.h

```diff
@@ -159,171 +264,135 @@ struct VariableViewImpl : public VariableImpl {
   Variable base;
 
   // The value of the version_counter at the time grad_fn was created. The
-  // _grad_fn field is stale if attr_version != version_counter.current_version()
-  int attr_version;
+  // grad_fn field is stale if attr_version !=
+  // version_counter.current_version()
+  uint32_t attr_version = 0;
```

Comment:
this doesn't really need to be initialized, and it's not true that it's always zero when a view is created. Adding initialization makes it more confusing

Reply:
Ok, I'll remove it.